### ISSUE TITLE
[a11y][WCAG 4.1.2]: Name of the link is not announced by the screen reader.

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#9919]
Observed Results:
--
When focus is on " preschool@seattle.gov" link, chrome vox is announcing as "Link opens in a new tab, press search + space to activate" in "Eligibility Test Program" page

Screencast: Chromebook + Chromevox[here]( https://drive.google.com/file/d/1kAeAtvJ3-RAXSpWbAG8rT1eQJSIPnC6m/view?usp=drive_link ) \
Screencast: Windows + NVDA[here]( https://drive.google.com/file/d/1B4biBe2vb7h7eIEeLWrf-YRpm5TpKLAl/view?usp=drive_link )

User Impact:
-----
Users who depend on assistive technologies like screen reader will face difficulty if name of the links are not announcing.

Expected Results:
--
When focus is on "preschool@seattle.gov" link, chromevox should announce as " preschool@seattle.gov" link opens in new tab, press search + space to activate"

Steps to Reproduce:
-------
0. Enable Chromevox using Ctrl+Alt+Z
1. Launch the URL[here]( https://ns.civiform.dev/programs )
2. Click "View and Apply" at "Eligibility test program".
3. Navigate to link at "How to apply" region and observe the screen reader verbalization over that link

WCAG Details:
--
[WCAG 4.1.2][Name, Role and Value]( https://www.w3.org/WAI/WCAG22/quickref/?versions=2.0#name-role-value )

Additional Information
--
NA

Environment Details:
--

Machine: chromebook 13.3/Windows \
Browser: Version 132.0.6834.208 (Official Build) (64-bit) \
Screenreader: Chromevox/NVDA 2024.4.2135031 \
Bug Filling Time: 15 mins.

**Comment by melanie-exygy** [comment]
This problem seems to be with links that are within markdown specifically.
